security/logging/common.py

```diff
@@ -0,0 +1,82 @@
+from threading import local
+
+from uuid import uuid4
+
+from contextlib import ContextDecorator
+
+from django.conf import settings as django_settings
+
+from security.config import settings
+
+
+class SecurityLogger(ContextDecorator, local):
```

Comment:
every logger exteds this class. Active loggers creates tree (property loggers) therefore you now parent loger. (For example output request inside input request)